files/fr/web/javascript/reference/statements/while/index.md

```diff
@@ -50,6 +50,22 @@ while (n < 3) {
 
 Une fois que la troisième itération est exécutée, la condition `n` < 3 n'est plus vérifiée et donc la boucle se termine.
 
+2. S'assurer que la function  imprime les nombres de 0 à 20 à l'écran en utilisant la boucle "while"
+
+```js
+const myWhileLoop = () =>{
+  let i = 0;
+  while (i< 20){
+  i++;
+  };
+    console.log(i);
+};
+```
+
+Dans l'exemple ci-haut, le code de la boucle s'exécutera encore et encore tant qu'une variable (i) est inférieure à 20 :
+
+**Attention**: => Si vous oubliez d'augmenter la variable utilisée dans la condition, la boucle ne se terminera jamais. Cela plantera votre navigateur.
+
```

Comment:
Plutôt que d'ajouter un autre exemple sous la forme d'un élément de liste numérotée, il faudrait plutôt utiliser le contenu à date de la version en anglais, avec un titre à part entière : https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/while#using_an_assignment_as_a_condition

Notes de correction : 
- Pas de doubles espaces (ici entre "function" et "imprime")
- Utilisation de `` ` `` comme symbole pour délimiter les mots-clés qui sont du code, par exemple `` `function` ``
- "ci-haut" > ci-avant
- (pinaillage typographique) : ajout de l'entité &amp;nbsp; devant les `:`
- Convention MDN : `**Attention :**` plutôt que `**Attention**:`

Reply:
Thank you , I see !